crates/l2/utils/config/mod.rs

```diff
@@ -34,3 +34,28 @@ pub fn read_env_file() -> Result<(), errors::ConfigError> {
 
     Ok(())
 }
+
+pub fn read_env_as_lines(
+) -> Result<std::io::Lines<std::io::BufReader<std::fs::File>>, errors::ConfigError> {
+    let env_file_name = std::env::var("ENV_FILE").unwrap_or_else(|_| ".env".to_string());
```

Comment:
Can't we use `unwrap_or` here?
```suggestion
    let env_file_name = std::env::var("ENV_FILE").unwrap_or(".env".to_owned());
```

Reply:
Maybe we can have a `DEFAULT_ENV_FILE_NAME` constant.